image_transport: raw    # image format
family: 36h11           # tag family name
size: 0.147             # tag edge size in meter
max_hamming: 0          # maximum allowed hamming distance (corrected bits)
use_pnp: false          # whether to use OpenCV PnP or original by Christian Rauch

# see "apriltag.h" 'struct apriltag_detector' for more documentation on these optional parameters
detector:
    threads: 1              # number of threads
    decimate: 1.0           # decimate resolution for quad detection
    blur: 0.1               # sigma of Gaussian blur for quad detection
    refine: 1               # snap to strong gradients
    sharpening: 1.0         # sharpening of decoded images
    critical_rad: 1.3090    # (75deg) Reject quads where pairs of edges have angles that are close to straight or close to 180 degrees.
    min_cluster_pixels: 128 # reject quads containing too few pixels
    debug: 0                # write additional debugging images to current working directory